ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -73,6 +75,7 @@ type ssh_core_options_type = {
     "Compression" ? string with match (SELF, '^(yes|delayed|no)$')
     "GSSAPIAuthentication" ? legacy_binary_affirmation_string
     "GSSAPICleanupCredentials" ? legacy_binary_affirmation_string
+    "GSSAPIKexAlgorithms" ? ssh_gssapikexalgorithms[]
```

Comment:
I think this should be `ssh_gssapikexalgorithms[1..]` as a missing value causes the component to fail at run-time.